templates/zerver/api/get-stream-topics.md

```diff
@@ -0,0 +1,76 @@
+# Get topics in a stream
+
+Get all the topics in a specific stream
+
+`GET {{ api_url }}/v1/users/me/<stream_id>/topics`
+
+## Usage examples
+<div class="code-section" markdown="1">
+<ul class="nav">
+<li data-language="python">Python</li>
+<li data-language="javascript">JavaScript</li>
+<li data-language="curl">curl</li>
+</ul>
+<div class="blocks">
+
+<div data-language="curl" markdown="1">
+
+```
+curl {{ api_url }}/v1/users/me/<stream_id>/topics \
+    -u BOT_EMAIL_ADDRESS:BOT_API_KEY
+```
+
+</div>
+
+<div data-language="python" markdown="1">
+
+{generate_code_example(python)|get-stream-topics|example}
+
+</div>
+
+<div data-language="javascript" markdown="1">
+More examples and documentation can be found [here](https://github.com/zulip/zulip-js).
+```js
+const zulip = require('zulip-js');
+
+// Download zuliprc-dev from your dev server, assuming it's in the local dir
+const config = {
+    zuliprc: 'zuliprc-dev',
+};
+
+zulip(config).then((client) => {
+    // Get all the topics in stream with ID 1
+    return client.streams.topics.retrieve({ stream_id: 1 });
+}).then(console.log);
+
+```
+</div>
+
+</div>
+
+</div>
+
+## Arguments
+
+**Note:** these arguments go in the endpoint's URL, not as a querystring.
```

Comment:
This line is misleading, "go in the endpoint's URL" and "as a querystring" are the same thing, right? Either way, I think it should be `**Note**: The following arguments are all URL query parameters.`. :)

Reply:
It isn't the same. An argument can be either in the URL's path (like in this case):

    chat.zulip.org/api/v1/users/me/THE_STREAM_ID_GOES_HERE/topics

Or as a querystring:

    chat.zulip.org/api/v1/some/endpoint?some_argument=some_value

These aren't interchangeable, and in this endpoint you should use the former.

If you think that a change in the phrasing can make this clearer I'll be happy to push an update :)

Reply:
@YagoGG: Oops my bad, sorry! Okay, well then let's just leave that `Note` part out. I feel like the examples should be enough for the reader to draw a distinction between how to pass in the argument (based on whether one is using Python, JS or curl).

Also, you were right that the endpoint doesn't technically take in any arguments but the Python code that interfaces with the endpoint *does* take in an argument and that's why I thought we should document it explicitly. Just thought I'd clarify! :)